pybossa/cloud_store_api/s3.py

```diff
@@ -195,8 +195,6 @@ def get_content_and_key_from_s3(s3_bucket, path, conn_name=DEFAULT_CONN,
             secret = app.config.get('FILE_ENCRYPTION_KEY')
         cipher = AESWithGCM(secret)
         content = cipher.decrypt(content)
-    if type(content) == bytes:
-        content = content.decode()
```

Comment:
Will the CSV and JSON file formate work with the removal?

Reply:
Under python3 everything comes as bytes and there's no easy way to distinguish between byte string(that requires decode) and non byte string. It seems we can keep this change and pass upon decoding error. I'll make an update to the code for this. Thanks!